Unix/build.mak

```diff
@@ -237,7 +237,7 @@ distclean:
 
 ifdef BUILD_TESTS
 tests:
-	( LD_LIBRARY_PATH=$(LIBDIR); export LD_LIBRARY_PATH; DYLD_LIBRARY_PATH=$(LIBDIR); export DYLD_LIBRARY_PATH; $(BINDIR)/nits -file:$(TMPDIR)/nitsargs.txt )
+	sudo sh -c "( LD_LIBRARY_PATH=$(LIBDIR); export LD_LIBRARY_PATH; DYLD_LIBRARY_PATH=$(LIBDIR); export DYLD_LIBRARY_PATH; $(BINDIR)/nits -file:$(TMPDIR)/nitsargs.txt )"
```

Comment:
not true. we don't need sudo to run tests. If you have previously run tests as sudo then you will either need to continue to use sudo, or blow away the output directory and start again as non-root.